stix_shifter_modules/splunk/stix_translation/results_translator.py

```diff
@@ -9,11 +9,9 @@ def __init__(self, options, dialect, base_file_path=None, callback=None):
     super().__init__(options, dialect, base_file_path, hash_type_lookup)
 
   def translate_results(self, data_source, data):
-    results = json.loads(data)
+    results = data
```

Comment:
minor issue. same as some other `results_translator.py`. this statement is unnecessary `results = data
`